.github/workflows/unix_linux.yml

```diff
@@ -0,0 +1,166 @@
+            BASE=Coreform-Cubit-2021.5%2B15962_5043ef39
+            CUBIT_BASE_NAME=Coreform-Cubit-2021.5
+            HDF5_PATH=/usr/local/HDF_Group/HDF5/1.12.0
+          fi
+
+          SUFFIX=Lin64
+          EXT=deb
+          echo "SED=sed" >> $GITHUB_ENV
+          echo "BUILD_SHARED_LIBS=ON" >> $GITHUB_ENV
+          echo "BUILD_STATIC_LIBS=OFF" >> $GITHUB_ENV
+          echo "system=linux" >> $GITHUB_ENV
+          echo "CUBIT_PATH=/opt/${CUBIT_BASE_NAME}" >> $GITHUB_ENV
+          echo "COREFORM_BASE_URL=${COREFORM_BASE_URL}/Linux/" >> $GITHUB_ENV
+          echo "HDF5_PATH=${HDF5_PATH}" >> $GITHUB_ENV
+
+          echo "OS=${{ matrix.os }}" >> $GITHUB_ENV
+          echo "CMAKE_ADDITIONAL_FLAGS=-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0" >> $GITHUB_ENV
+
+          echo "CUBIT_PKG=${BASE}-${SUFFIX}.${EXT}" >> $GITHUB_ENV
+          echo "CUBIT_SDK_PKG=${BASESDK}-${SUFFIX}.tar.gz" >> $GITHUB_ENV
+          echo "CUBIT_BASE_NAME=${CUBIT_BASE_NAME}" >> $GITHUB_ENV
+
+          echo "CURRENT=$(pwd)" >> $GITHUB_ENV
+          echo "SCRIPTPATH=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          echo "PLUGIN_ABS_PATH=$GITHUB_WORKSPACE/.." >> $GITHUB_ENV
+          echo "FOLDER_PKG=$GITHUB_WORKSPACE/pkg" >> $GITHUB_ENV
+
+          echo "" >> ${HOME}/.bash_profile
+          echo "source $GITHUB_WORKSPACE/scripts/unix_share_build.sh" >> $HOME/.bash_profile
+      
+      - name: Initial setup
+        shell: bash -l {0}
+        run: |
+          ${system}_install_prerequisites
+ 
+      - name: Downloading packages
+        shell: bash -l {0}
+        run: |
+          mkdir -p $FOLDER_PKG
+          cd ${FOLDER_PKG}
+          curl -L ${COREFORM_BASE_URL}${CUBIT_PKG} --output ${CUBIT_PKG}
+          if [ "${{ matrix.cubit }}" = "17.1.0" ]; then
+            curl -L ${COREFORM_BASE_URL}${CUBIT_SDK_PKG} --output ${CUBIT_SDK_PKG}
+          fi
+          mkdir ${SCRIPTPATH}/release
+
+      - name: Cubit setup
+        shell: bash -l {0}
+        run: |
+          ${system}_setup_cubit ${{ matrix.cubit }}
+
+      - name: Build HDF5
+        shell: bash -l {0}
+        run: |
+          ${system}_build_hdf5
+
+      - name: Build MOAB
+        shell: bash -l {0}
+        run: |
+          build_moab
+
+      - name: Build DAGMC
+        shell: bash -l {0}
+        run: |
+          build_dagmc
+
+      - name: Build plugin
+        shell: bash -l {0}
+        run: |
+          build_plugin 
+
+      - name: Prepare package
+        shell: bash -l {0}
+        run: |
+          ${system}_build_plugin_pkg ${{ matrix.cubit }}
+
+      - if: github.event_name != 'release'
+        name: Upload artifact for CI
+        uses: actions/upload-artifact@v2
+        with:
+          name: svalinn-plugin_${{ matrix.os }}-${{ matrix.os_version }}_cubit_${{ matrix.cubit }}.tgz
+          path: ${{ github.workspace }}/release/svalinn-plugin_${{ matrix.os }}-${{ matrix.os_version }}_cubit_${{ matrix.cubit }}.tgz
```

Comment:
```suggestion
          path: ${{ github.workspace }}/release/svalinn-plugin_${{ matrix.os }}-${{ matrix.os_version }}_cubit_${{ matrix.cubit }}.tgz
          if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn` 
```

Reply:
this will allow to be aware when we have a problem founding the tgz file

Reply:
Thanks, this is a nice check.